### ISSUE TITLE
Throws an error when using ES6 template literals

### ISSUE THREAD

**Issue by pensierinmusica** [shkuznetsov/gulp-minify-inline#18]
`gulp-minify-inline` (v0.2.1) throws an error when using [ES6 template literals](http://es6-features.org/#StringInterpolation).

This works fine:

```js
var win = window.open('https://youtu.be/' + this.videoId, '_blank');
```

This throws an error:

```js
var win = window.open(`https://youtu.be/${this.videoId}`, '_blank');
```

```sh
SyntaxError: Unexpected character '`'
    at JS_Parse_Error.get (eval at <anonymous> (.../node_modules/gulp-minify-inline/node_modules/uglify-js/tools/node.js:27:1), <anonymous>:86:23)
```

It looks like the error comes from `uglify-js`, which doesn't seem to support ES6 syntax. Shall we switch to `uglify-es` instead? Do you want me to send a PR?

Cheers!

**Comment by shkuznetsov** [comment]
Hi @pensierinmusica 

Yeah, looks like I have not updated the dependencies for far too long. I am keen to upgrade, but would want to better understand backward compatibility situation. Like will it run on pre-harmony versions of node? Also `uglify-es` looks much less mature than `uglify-js`. Shall we make it optional maybe?

**Comment by pensierinmusica** [comment]
@shkuznetsov mmm if I understand correctly they're just two different versions of the same software:

- `uglify-js` - https://github.com/mishoo/UglifyJS2
- `uglify-es` - https://github.com/mishoo/UglifyJS2/tree/harmony

The `package.json` file of `uglify-es` says it supports Node `>=0.8.0`, so if that is correct it should work on pre-harmony.

When you run the tests does it give you any issue?

Hope this helps!

**Comment by shkuznetsov** [comment]
Oh you're so right, I'm sooooo lost the thread, sorry. Yes please if you've got PR — please feel free to drop in. Otherwise I'll have a look in a few days.

**Comment by shkuznetsov** [comment]
Fixed, bumped major version (v1.0.0) due to underlying plugins API changes

**Comment by pensierinmusica** [comment]
Awesome!
